cypress/integration/1-getting-started/todo.spec.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
+
+// Welcome to Cypress!
+//
+// This spec file contains a variety of sample tests
+// for a todo list app that are designed to demonstrate
+// the power of writing tests in Cypress.
+//
+// To learn more about how Cypress works and
+// what makes it such an awesome testing tool,
+// please read our getting started guide:
+// https://on.cypress.io/introduction-to-cypress
+
```

Comment:
We should remember to delete the boilerplate tests.

Reply:
Good call. We kind of abandoned Cypress for Router. Are we traitors?